app/config/mq-config.js

```diff
@@ -48,6 +48,20 @@ const mqSchema = joi.object({
     address: joi.string().default('request-sbi'),
     username: joi.string(),
     password: joi.string()
+  },
+  responseStandardsQueue: {
+    name: joi.string().default('ffc-sfi-response-standards'),
+    address: joi.string().default('response-standards'),
+    username: joi.string(),
+    password: joi.string(),
+    type: joi.string()
+  },
+  responseCalculateQueue: {
+    name: joi.string().default('ffc-sfi-response-calculate'),
+    address: joi.string().default('response-calculate'),
```

Comment:
Suggest removing the default values for `address`.  With a default value the joi validation always passes even if local env vars aren't set.  This means that the app starts fine and only fails when it tries to use service bus.

It would be better if it failed fast and devs got immediate feedback that something is missing.

`name` isn't really required but no harming having it.

Reply:
Agree